pkg/snapshot/snapshotter/snapshotter.go

```diff
@@ -569,26 +591,54 @@ func (ssr *Snapshotter) snapshotEventHandler(stopCh <-chan struct{}) error {
 			if err != nil {
 				return err
 			}
+			latestFullSnapshot, latestDeltaSnapshotList, err := miscellaneous.GetLatestFullSnapshotAndDeltaSnapList(ssr.store)
+			if err == nil {
+				if err = heartbeat.UpdateDeltaSnapshotLeases(ssr.logger, latestFullSnapshot, latestDeltaSnapshotList, ssr.k8sClientset); err != nil {
+					ssr.logger.Warnf("1 Failed to update delta snapshot lease with error: %v", err)
```

Comment:
Is there a particular reason for the `1` at the beginning of the error log? I see this in the subsequent logs as well, like 1, 2 and 3. Can you please explain this? Maybe it gives some extra information for debugging live issues, or it was just used for testing your PR perhaps?